poll/async/get.js

```diff
@@ -80,3 +85,16 @@ function decoratedPoll (rawPoll, msgs = []) {
 
   return poll
 }
+
+function decoratePosition ({position: rawPosition, poll: rawPoll}) {
+  var position = getMsgContent(rawPosition)
+  var poll = getMsgContent(rawPoll)
+
+  var newPosition = Object.assign({}, rawPosition)
```

Comment:
If this intention here is to clone, then we need to go deeper - remember that position has `positionDetail` nested in it

Reply:
I think mutating here is ok given we are decorating - as in not overriding the key+value, just extending ... but the if we're being 'functional' mm

Reply:
:heavy_multiplication_x: leaving this for now